app/static/currency.js

```diff
@@ -0,0 +1,54 @@
+function getCurrency(baseCurrency, dateToday) {
+    const showCurrencyElement = document.getElementById("showCurrency")
+    const currencyViewElement = document.getElementById("currencyView");
+    const currencyButtonElement = document.getElementById("currencyButton");
+    currencyViewElement.innerHTML = "";
+
+    function getResponse(yourUrl) {
+        var httpRequest = new XMLHttpRequest();
+        httpRequest.open("GET", yourUrl, false);
+        httpRequest.send(null);
+        return httpRequest.responseText;
```

Comment:
Prefer managing the request asynchronously :)

Reply:
I don’t know anything about JS so this has been really helpful, thank you for the feedback ❤️